docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -342,10 +343,41 @@ paths:
           description: Limit the results to annotations tagged with the specified value.
           required: false
           type: string
+        - name: tags
+          in: query
+          description: Alias of `tag`.
+          required: false
+          type: string
         - name: any
           in: query
-          description: |
-            Limit the results to annotations in which one of a number of common fields contain the passed value.
+          description: Limit the results to annotations whose quote, tags, text or url fields contain this key word.  
+          required: false
+          type: string
+        - name: _separate_replies
+          in: query
+          description: Returns a seperate set of annotations and their replies.
+          required: false
+          type: string
+        - name: group
+          in: query
+          description: Limit the results to this group of annotations.
+          required: false
+          type: string
+        - name: quote
+          in: query
+          description: Limit the results to annotations that contain this text in their textual body.
+          required: false
+          type: string
+        - name: references
+          in: query
+          description: A list of parent annotation ids by which resulting annotations are
```

Comment:
Maybe I'm misunderstanding what references is but I thought it was a list of parent ids that the annotation was replying to so that description you have there doesn't make sense. I hear what you are saying though-about the description being unclear and it looks like no you can't pass a list, only a single value.